pkg/render/monitor/monitor.go

```diff
@@ -186,10 +187,16 @@ func (mc *monitorComponent) Objects() ([]client.Object, []client.Object) {
 
 	// Create role and role bindings first.
 	// Operator needs the create/update roles for Alertmanager configuration secret for example.
-	toCreate = append(toCreate,
-		mc.operatorRole(),
-		mc.operatorRoleBinding(),
-	)
+
+	roles := mc.operatorRole()
```

Comment:
Can be written as `toCreate = append(toCreate, mc.operatorRole()...)`
same for the loop below.